Create CSS Entries for translateX & translateY and Update translate (#576)
Codecademy/docs: content/css/concepts/transform-functions/terms/translateY/translateY.md

---
Title: 'translateY()'

> Description: 'Translates an element along the Y axis.'

Reviewer: ```suggestion
Description: 'Translates an element along the Y-axis.'
```

> Description: 'Translates an element along the Y-axis.'

Reviewer 2: ```suggestion
Description: 'Moves an element vertically along the Y-axis.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Functions'
  - 'Positioning'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
  - 'paths/full-stack-engineer-career-path'
---

> The `translateY` function translates an element along the Y axis.

Reviewer: ```suggestion
The `translateY` function translates an element along the vertical Y-axis.
```

> The `translateY` function translates an element along the vertical Y-axis.

Reviewer 2: ```suggestion
The `translateY()` function moves an element along the vertical Y-axis.
```

## Syntax

```pseudo
transform: translateY(<value>);
```

The required `<value>` can be one of the following:

- Length value: `100px`, `1.5em`
- Percentage value: `25%`, `50%`

> A positive value moves the element down the vertical axis while a negative value will move an element up.

Reviewer: ```suggestion
A positive value moves the element down the vertical Y-axis, while a negative value will move an element up.
```

## Example 1

> Move the `.box` element `100px` down:

Reviewer 2: ```suggestion
In the example below, a `.box` element is moved down the Y-axis by `100px`:
```

```css
.box {
  transform: translateY(100px);
}
```

## Example 2

> Move the `.box` element `10em` up:

Reviewer 2: ```suggestion
In this example, the element is moved up the Y-axis by `-10em`, relative to the height of its direct parent element.
```

```css
.box {

>   transform: translateX(-10em);

Reviewer: ```suggestion
  transform: translateY(-10em);
```
